app/src/main/java/de/westnordost/streetcomplete/quests/bbq_fuel/AddBBQFuel.kt

```diff
@@ -0,0 +1,39 @@
+package de.westnordost.streetcomplete.quests.bbq_fuel
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddBBQFuel : OsmFilterQuestType<BBQFuel>() {
```

Comment:
Not sure what the convention in this repository is, but I always prefer "camel-casing" acronyms, i.e. `AddBbqFuel`, `BbqFuel`, etc. instead of `AddBBQFuel`, `BBQFuel`, etc.

Reply:
Hast recht, in den ATM-Quests ist es auch camel-case. Habs geändert.